libr/util/lib.c

```diff
@@ -303,19 +303,80 @@ R_API int r_lib_open_ptr (RLib *lib, const char *file, void *handler, RLibStruct
 
 	return ret;
 }
+#if __WINDOWS__ && !defined(__CYGWIN__)
+static char *utf16_to_utf8 (const wchar_t *wc) {
+	char *rutf8;
+	int csize;
+	
+	csize = WideCharToMultiByte (CP_UTF8, 0, wc, -1, NULL, 0, NULL, NULL);
+	if (csize > 0) {
+		rutf8 = malloc (csize);
```

Comment:
Check for null and you can assign and define csize and rutf8 in yhe same line and reduce locs while making it more readablr